### PR TITLE
Fix ldd_test to test shared library dependency checking

### PR DESCRIPTION
**Pull request by rminnich**
Signed-off-by: Ronald G. Minnich <rminnich@gmail.com>

### CODE REVIEW

**Comment by rminnich** [comment]
I gotta take out some of that 'this sucks' language, so this will have another pass.

**Comment by rminnich** [comment]
language now cleaned up, ptal

File: pkg/ldd/ldso_linux.go [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package uroot
+
+import (
+	"fmt"
+	"path/filepath"
+	)
```
Comment (bkochendorfer) [inline pkg/ldd/ldso_linux.go]: I think you need to run `go fmt` on this file.

**Comment by rminnich** [comment]
done

**Pull request merged**